docs/admin/configuration.md

```diff
@@ -3,8 +3,10 @@ Configuration
 
 ## bpfd
 
-bpfd expects a configuration file to be present at `/etc/bpfd.toml`.
+bpfd expects a configuration file to be present at `/etc/bpfd/bpfd.toml`.
 If no file is found, defaults are assumed.
+The `scripts/setup.sh` commands will copy a default version of the file to the correct location
+from `scripts/bpfd.toml`, which can then be overwritten if needed.
```

Comment:
So now we essentially will never use the rust defaults right?

Reply:
If the script is run, we essentially will never use the rust defaults. But we can't be guaranteed the script is run.